src/modules/wifi/sniffer.cpp

```diff
@@ -414,7 +414,7 @@ void sniffer_setup() {
       esp_wifi_set_promiscuous_rx_cb(sniffer);
     }
 
-    #if defined(CARDPUTER)
+    #if defined(CARDPUTER) || defined(T_EMBED)
```

Comment:
Maybe `if defined(HAS_KEYBOARD)` is correct?

Reply:
in fact, T_EMBED uses a different ESC btn, and using the checkPrev to escape won´t work with this device, 

but it need to include T-Deck in this logic by changing cardputer for HAS_KEYBOARD